src/types/api.js

```diff
@@ -33,6 +33,12 @@
  * @typedef {TextQuoteSelector | TextPositionSelector | RangeSelector} Selector
  */
 
+/**
+ * @typedef Target
+ * @prop {string} target.source
+ * @prop {Selector[]} [target.selector]
```

Comment:
```suggestion
 * @prop {string} source
 * @prop {Selector[]} [selector]
```

I'm surprised this didn't generate an error but in any case, the parent property name is only needed for "inline" typedefs inside a parameter list or other typedef.

Reply:
Must have been a refactor copy paste error, my mistake. And yes, I agree, the error would have been nice.